test/extended/setup.sh

```diff
@@ -6,12 +6,12 @@
 function os::test::extended::focus {
   if [[ $# -ne 0 ]]; then
     echo "[INFO] Running custom: $@"
-    tests=$(TEST_REPORT_DIR= TEST_OUTPUT_QUIET=true ${EXTENDEDTEST} --ginkgo.dryRun --ginkgo.noColor "$@" | col -b | grep -v "35mskip0m" | grep "1mok0m" | wc -l)
+    tests=$(TEST_REPORT_DIR= TEST_OUTPUT_QUIET=true ${EXTENDEDTEST} --ginkgo.dryRun --ginkgo.noColor "$*" | col -b | grep -v "35mskip0m" | grep "1mok0m" | wc -l)
     if [[ "${tests}" -eq 0 ]]; then
       echo "[ERROR] No tests would be run"
       exit 1
     fi
-    ${EXTENDEDTEST} "$@"
+    ${EXTENDEDTEST} "$*"
```

Comment:
We can see the effect of `$*` *etc* with a simple script:
```
With args:
'one' 'two 2' 'three 3' '4' '5'
Using $*:
'one' 'two' '2' 'three' '3' '4' '5' 
Using "$*":
'one two 2 three 3 4 5' 
Using $@:
'one' 'two' '2' 'three' '3' '4' '5' 
Using "$@":
'one' 'two 2' 'three 3' '4' '5' 
```

I believe we want `"$@"` here, as well as above.